migrations/sql/V2019.06.26.10.30__add_code_required_report_tables.sql

```diff
@@ -0,0 +1,154 @@
+CREATE TABLE mine_report_due_date_type (
+    mine_report_due_date_type         character varying(3)                    PRIMARY KEY,
+    description                       character varying(60)                   NOT NULL   ,
+    active_ind                        boolean DEFAULT true                    NOT NULL   ,
+    create_user                       character varying(60)                   NOT NULL   ,
+    create_timestamp                  timestamp with time zone DEFAULT now()  NOT NULL   ,
+    update_user                       character varying(60)                   NOT NULL   ,
+    update_timestamp                  timestamp with time zone DEFAULT now()  NOT NULL
+);
+
+COMMENT ON TABLE mine_report_due_date_type is 'Types of due dates available for mine reports.';
+ALTER TABLE mine_report_due_date_type OWNER TO mds;
+
+CREATE TABLE mine_report_submission_status_code (
+    mine_report_submission_status_code  character varying(3)   PRIMARY KEY DEFAULT 'MIA',
```

Comment:
Apologies if this is a dumb question, but should there be a default value on a Primary Key?

Reply:
@JustinMacaulay Should this be in the backend not in the db?

Reply:
removed the default.